docs/source/scripts_and_their_functions.rst

```diff
@@ -100,6 +100,14 @@ required. The Buffer Delay Node is primarily intended for delaying implicitly
 timed documents for resynchronisation. Use ``ebu-run`` to start this script, for
 example ``ebu-run --admin.conf=ebu_tt_live/examples/config/buffer_delay.json``
 
+DeDuplicator Node
+-----------------
+This node addresses the issues raised upon ReSequencing, where ``style`` and
```

Comment:
Actually it will deduplicate any input, it's just that ReSequencing might cause such issues.